.travis.yml

```diff
@@ -13,8 +13,5 @@ before_script:
 script:
   - vendor/bin/phpunit
 
-after_script:
-  - vendor/bin/coveralls
-
-notifications:
-  email: matthiasnoback@gmail.com
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Okey. I reacted to the fact of running a remote script. 

But like I said, this is fine.